Chart.yaml

```diff
@@ -3,9 +3,10 @@ apiVersion: v2
 description: Helm chart for Orchestrate
 name: orchestrate
 type: application
-version: 1.0.8
+version: 2.0.0
 home: https://docs.orchestrate.consensys.net/
 icon: https://docs.orchestrate.consensys.net/en/stable/images/logo.svg
 maintainers:
   - name: christian.tran
-appVersion: 21.1.12
+  - name: sylvain.decourval
+appVersion: v21.12.1
```

Comment:
Should we be adding a dependency re the database here too similar to https://github.com/ConsenSys/quorum-kubernetes/blob/master/dev/helm/charts/blockscout/Chart.yaml#L24 

It's not clear where the database is or how its created at the moment

Reply:
One database implementation (Postgres) is proposed in the calling chart here https://github.com/ConsenSys/orchestrate-kubernetes